src/Documents/Operations/RevisionsCollectionConfiguration.ts

```diff
@@ -0,0 +1,14 @@
+import {SetupDocumentBase} from "../SetupDocumentBase";
+import {ObjectUtil} from "../../Utility/ObjectUtil";
+
+export class RevisionsCollectionConfiguration extends SetupDocumentBase {
```

Comment:
While we're at it, do we need client configuration commands/operations to use`SetupDocumentBase`?

Reply:
no - it is used only when we have props with some custom serialization logic. Like:

```
public class PutConnectionStringResult {

    @JsonProperty("ETag")   <-- in node.js we have to use SetupDocumentBase
    private Long etag;

    public Long getEtag() {
        return etag;
    }

    public void setEtag(Long etag) {
        this.etag = etag;
    }
}
```